nilearn/regions/tests/test_region_extractor.py

```diff
@@ -155,6 +155,12 @@ def test_region_extractor_fit_and_transform():
     assert_not_equal(extract_ratio.regions_img_, '')
     assert_true(extract_ratio.regions_img_.shape[-1] >= 9)
 
+    # No region for too high FWHM
```

Comment:
I am wandering whether we should test with
 -   ```float``` as input to smoothing_fwhm
- low min_region_size_in_mm3 taking data resolution example same as yours.

Does this sounds good ?